src/modules/0.0.8/aws_ec2/migration/1654893453604-aws_ec2.ts

```diff
@@ -14,26 +14,9 @@ export class awsEc21653932287040 implements MigrationInterface {
         await queryRunner.query(`ALTER TABLE "registered_instance" ADD CONSTRAINT "FK_bdcacf2c01109dd6ad3ffab7a83" FOREIGN KEY ("target_group") REFERENCES "target_group"("target_group_name") ON DELETE CASCADE ON UPDATE NO ACTION`);
         await queryRunner.query(`ALTER TABLE "instance_security_groups" ADD CONSTRAINT "FK_fa3c179d5090cb1309c63b5e20a" FOREIGN KEY ("instance_id") REFERENCES "instance"("id") ON DELETE CASCADE ON UPDATE CASCADE`);
         await queryRunner.query(`ALTER TABLE "instance_security_groups" ADD CONSTRAINT "FK_b3b92934eff56d2eb0477a1d27f" FOREIGN KEY ("security_group_id") REFERENCES "security_group"("id") ON DELETE CASCADE ON UPDATE CASCADE`);
-        await queryRunner.query(`
-            create or replace function check_target_group_instance(_target_group_name text) returns boolean
-            language plpgsql security definer
-            as $$
-            declare
-                _target_group_type target_group_target_type_enum;
-            begin
-                select target_type into _target_group_type
-                from target_group
-                where target_group_name = _target_group_name;
-                return _target_group_type = 'instance';
-            end;
```

Comment:
what is this used for? hmm should we have it `iasql_functions`?

Reply:
or have the migration (re)generation automatically regenerate this so we don't add a manual step?

Reply:
this is a constraint to ensure that you are only allow to assign a target group of type `instance` to a ec2 instance. For now it has to be done manually, we do not have other way to add this automatically on the migrations generation and cannot be expressed with typeorm.

Reply:
got it 👍